### DISCUSSION TITLE
Failed to connect to relay server error

### DISCUSSION THREAD

**Discussion by AlexanderWolfZhang** [rustdesk/rustdesk#8420] [General]
Hey guys
I recently deploy a rustdesk server via docker on ubuntu server 24.04LTS, everything seems works fine, but when I try to deploy the client using offical powershell script I encounter some problems
First I set up one client like this:
![image](https://github.com/rustdesk/rustdesk/assets/24367379/029a6e28-7be5-48c2-963f-3e546abd707e)
Note: rustdesk.cleanlamt.cn and relyrustdesk.CleanlAMT.cn are point to the same ip addresses
Then I export the config string, and put it in the client deployment script, remove any "=" at the start or end of the string, and ran it on another computer, after it was done, the network config looks like this:
![image](https://github.com/rustdesk/rustdesk/assets/24367379/d9ab4844-430f-4d01-b9b8-f8a5af3e1135)
![image](https://github.com/rustdesk/rustdesk/assets/24367379/100cd7c4-184a-42b1-b56f-3dc91dd44157)
The relay server were left blank, and the client says it was ready, but when I try to connect to the client computer, It gives me an "Failed to connect to relay server" error:
![image](https://github.com/rustdesk/rustdesk/assets/24367379/2061eece-c1a8-4882-9a66-a6207c71387e)
 then I tried to manully input the relay server in client network setting, it works normally
How do I fix this problem?

**Comment by rustdesk** [comment]
Nobody will answer you here, please go to https://discord.com/invite/nDceKgxnkV
